gsi_wdl_tools/workflow_info.py

```diff
@@ -61,15 +62,32 @@ def outputs(self):
             elif isinstance(output, WDL.Env.Binding):
                 val = output_descriptions.get(output.name)
                 if val is not None:
-                    description = val
+                    description = val.get("description", "")
```

Comment:
I am currently getting an exception with an existing workflow has not been updated to have vidarr_labels:
```
Traceback (most recent call last):
  File "gsi-wdl-tools/bin/../scripts/generate_markdown_readme.py", line 111, in <module>
    for output in info.outputs:
  File "gsi_wdl_tools/workflow_info.py", line 65, in outputs
    description = val.get("description", "")
AttributeError: 'str' object has no attribute 'get'
```